package customExceptions

internal class CustomerNotFoundException: Exception("Customer does not exist")